src/features/dashboard/DashboardPage.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import Dots from '../../common/components/Icons/Dots'
+import TransactionItem, { TTransactionItemProps } from './TransactionItem'
+import PortfolioColumn from './PortfolioColumn'
+import PortfolioItem, { TPortfolioItemProps } from './PortfolioItem'
+import NFTCard, { INFTCompactCardProps } from '../../common/components/NFTCard'
+import Notification from './Notification'
+import LinkSection from './LinkSection'
+import dayjs from 'dayjs'
+import smallImage from '../../common/assets/images/mock/small-image.png'
+import image from '../../common/assets/images/nft-card-placeholder.png'
+import { formatNumber } from '../../common/utils/format'
+
+const date = dayjs('2021-04-04')
+
+const walletBalance = 32000
+const currencyName = 'PSTL'
```

Comment:
@CrazyOptimist I think `PSL` is correct instead of `PSTL`

Reply:
gotcha!